src/typeGuards.ts

```diff
@@ -20,5 +20,5 @@ export function isInGroupSuggestion(item: Item): item is InGroupSuggestion {
 }
 
 export function isCustomSection(config: UserDefinedSection): config is CustomSection {
```

Comment:
Can we add other type guards as this one for `isAutocompleteSection` and `isRecommendationSection`?

Reply:
We can but why? If we are not using them, why are we adding them? For consumers of the library?

Reply:
Add type guards